stream/src/ngx_stream_lua_kong_module.c

```diff
@@ -323,3 +323,25 @@ void **ssl_conn)
 #endif
 }
 
+
+int
+ngx_stream_lua_kong_ffi_get_request_ssl(ngx_stream_lua_request_t *r, void **ssl_conn)
+{
+#if (NGX_SSL)
+    if (ssl_conn == NULL) {
+        return NGX_ABORT;
+    }
+
+    ngx_connection_t *c = r->connection;
+
+    if (c && (c->ssl) && (c->ssl->connection)) {
+        *ssl_conn = c->ssl->connection;
+        return NGX_OK;
+    }
+
+    return NGX_ERROR;
+#else
+    return NGX_ABORT;
+#endif
+}
```

Comment:
Are KAG-5388 and KAG-5473 require this API for the stream module? If not, we could remove it to make the codebase smaller.